metricbeat/etc/fields.yml

```diff
@@ -8,11 +8,83 @@ defaults:
   ignore_above: 1024
 
 sections:
+  - ["common", "Common fields"]
   - ["redis-info", "Redis Info Stats"]
+  - ["mysql-status", "MySQL Status"]
+
+common:
+  type: group
+  description: >
+    Contains common fields available in all event types.
+  fields:
+    - name: beat.name
+      description: >
+        The name of the Beat sending the log messages. If the shipper name is set
+        in the configuration file, then that value is used. If it is not set,
+        the hostname is used.
+
+    - name: beat.hostname
+      description: >
+        The hostname as returned by the operating system on which the Beat is
+        running.
+
+    - name: "@timestamp"
+      type: date
+      required: true
+      format: YYYY-MM-DDTHH:MM:SS.milliZ
+      example: 2015-01-24T14:06:05.071Z
+      description: >
+        The timestamp when the log line was read. The precision is in
+        milliseconds. The timezone is UTC.
+
+    - name: response_time
+      type: long
+      required: true
+      description: >
+        Event Reponse time in nano seconds
+mysql-status:
+  type: group
+  description: >
+    MySQL Stats
+  fields:
+    - name: mysql-status
```

Comment:
> Agree for mysql_status.

@ruflin woops, I merged without that change. But you can address it in a separate PR.